tests/core-replication-state.js

```diff
@@ -377,17 +383,21 @@ function setPeerWants(crs, peerId, bits) {
 
 /**
  * Wait for update event with a timeout
- * @param {CoreReplicationState} crs
+ * @param {CoreSyncState} crs
```

Comment:
maybe `state` or `coreSyncState` would be better, but at the very least update the abbreviation.

```suggestion
 * @param {CoreSyncState} css
```